crates/jstz_api/src/stream/readable/mod.rs

```diff
@@ -38,9 +44,28 @@ impl NativeClass for ReadableStreamClass {
         context: &mut Context<'_>,
     ) -> JsResult<Self::Instance> {
         let underlying_source =
-            Option::<UnderlyingSource>::try_from_js(args.get_or_undefined(0), context)?;
-        let _ = underlying_source;
-        todo!()
+            Option::<UnderlyingSource>::try_from_js(args.get_or_undefined(0), context)?
+                .unwrap_or_else(|| todo!());
+        let queuing_strategy =
+            Option::<QueuingStrategy>::try_from_js(args.get_or_undefined(1), context)?
+                .unwrap_or_default();
+
+        // TODO Perform ! InitializeReadableStream(this).
```

Comment:
Can these todos stay here?

Reply:
Yes, I partially implemented the `ReadableStream` constructor to allow testing `UnderlyingSource` and `QueuingStrategy`, but since the implementation is not complete I added `todo!` and `TODO` comments (with `TODO` comments for everything that should be added before the parts used for the tests). Whoever will eventually implement this will probably know that these steps are missing, but I think having a comment there doesn't hurt